frontend/lang/compiled/en.json

```diff
@@ -1 +1,2 @@
-{}
+{
+}
```

Comment:
Can you restore the `/lang/compiled/*` files as they were before? I will fix the `.gitignore` configuration in `develop` directly.